gm4_mountaineering/data/gm4_mountaineering/tags/blocks/climbable_blocks.json

```diff
@@ -0,0 +1,7 @@
+{
+    "values":[
+    "minecraft:stone",
+    "minecraft:dirt",
+    "minecraft:grass_block"
+    ]
+   }
```

Comment:
missing newline

Stone variants should also be included here, you can use `#minecraft:base_stone_overworld` to cover them all. I feel like the ice variants should also be included, maybe things like ores as well in case they're in the wall. Probably some more that I missed